src/common/asb/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+project(asb)
+
+add_library(asb STATIC
+    Asb.c)
+
+target_compile_options(asb PRIVATE -Wno-psabi)
+
+target_include_directories(asb PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${PLATFORM_INC_DIR})
+
+target_link_libraries(asb PRIVATE 
+    logging 

```

Comment:
Same here, can we make `logging`, `commonutils` PUBLIC so that the downstream target doesn't have to include the libraries?